### PR TITLE
Add ATOODS and CCOODS

### PR DESCRIPTION
**Pull request by srp3rd**

### CODE REVIEW

File: sal_interfaces/SALSubsystems.xml [hunk 1]
```diff
@@ -163,6 +163,52 @@
...
+  <ActiveDevelopers>Steve Pietrowicz</ActiveDevelopers>
+  <PrincipalCSCOwner>Michael Reuter</PrincipalCSCOwner>
+  <Github>https://github.com/lsst-dm/ctrl_oods</Github>
+  <JenkinsTestResults>NA</JenkinsTestResults>
+  <LSSTPoC>Michael Reuter</LSSTPoC>
+  <CSCDocs>NA</CSCDocs>
+  <ProductOwner>Michael Reuter</ProductOwner>
+  <RelatedDocuments>Don't know but there is</RelatedDocuments>
+  <SoftwareLanguage>Python</SoftwareLanguage>
+  <RuntimeLanguages>CPP,LabVIEW,Java,PyDDS,Python</RuntimeLanguages>
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:182]: If you are using SalObj for this, your runtime is PyDDS.

File: sal_interfaces/SALSubsystems.xml [hunk 2]
```diff
@@ -163,6 +163,52 @@
...
```
Comment (mareuter) [inline sal_interfaces/SALSubsystems.xml:205]: Ditto.

File: sal_interfaces/SALSubsystems.xml (hunk 1 above)
Comment (srp3rd) [reply sal_interfaces/SALSubsystems.xml]: Fixed

File: sal_interfaces/SALSubsystems.xml (hunk 2 above)
Comment (srp3rd) [reply sal_interfaces/SALSubsystems.xml]: Fixed

**Pull request merged**